examples/Bunch/include/simulation_defines/param/physicalConstants.param

```diff
@@ -26,6 +26,11 @@ namespace picongpu
 {
     const float_64 PI = 3.141592653589793238462643383279502884197169399;
 
+    /** Discriminant used for calculations that want to separate between
```

Comment:
my idea was: the discriminant discriminates between the relativistic and the classical regime.